docs/README.md

```diff
@@ -0,0 +1,64 @@
+# undici
```

Comment:
This file seems a duplication of the project README. is this correct? Can we avoid that duplication?

Reply:
Yeah this is an unfortunate part of using these site generators. They sorta need everything to be included in the same directory. We could write some sort of copy script thats included in the build pipeline. From my cursory review of other options (docusaurus and vuepress), docsify is the simplest

Reply:
Check out https://github.com/mercurius-js/mercurius configuration. It does not duplicate the README.

Reply:
oh smart - but then the site file lives in repo root. If we are cool with that I'll do it here too

Reply:
Better than duplicating!